src/mi2.ts

```diff
@@ -59,6 +59,8 @@ export class MI2 extends EventEmitter implements IDebugger {
 	load(cwd: string, target: string, targetargs: string, group: string[]): Thenable<any> {
 		if (!nativePath.isAbsolute(target))
 			target = nativePath.join(cwd, target);
+		if(this.cobcpath === "docker" && this.gdbpath === "docker" && process.platform === "win32")
```

Comment:
Passing of `args.docker` to `new MI2` and check `docker !== undefined` might be better.